scripts/test.sh

```diff
@@ -6,9 +6,14 @@ if [ -z "$TEST_GREP" ]; then
 fi
 
 node="node"
+runSequentially=""
 
 if [ "$TEST_DEBUG" ]; then
    node="node --inspect --debug-brk"
```

Comment:
Do we need the `--inspect` flag? According to Jest documentation we don't? See https://facebook.github.io/jest/docs/troubleshooting.html#tests-are-failing-and-you-don-t-know-why.